monocle/static/js/main.js

```diff
@@ -357,8 +357,20 @@ function getWorkers() {
     });
 }
 
-var map = L.map('main-map', {preferCanvas: true}).setView(_MapCoords, 13);
+var params = {};
+window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m, key, value) {
+  params[key] = value;
+});
 
+if(params.lat && params.lon){
```

Comment:
I need to add a check here to prevent lat=abc&lon=edf from passing. No time, I'll add something tomorrow